src/index.ts

```diff
@@ -120,7 +121,10 @@ const handleMessage = async (message: Message) => {
   }
   const msg = message.partial ? await message.fetch() : message;
 
-  const channelId = msg.channel.id;
+  const channel = msg.channel;
+  const channelId = channel.isThread()
```

Comment:
I tested the other message handlers and they seemed to work, but this could add them as well in threads and do we want that? This could be a breaking change

Reply:
This seems fine to me, treating threads as if they came in via the parent channel makes intuitive sense to me